website/templates/search.mako

```diff
@@ -132,6 +132,18 @@
         <br>
         <!-- /ko -->
     </script>
+    <script type="text/html" id="file">
+        <h4><a href="{{ deep_url }}">{{ name }}</a> (<span data-bind="if: is_registration">Registration </span>File)</h4>
+        <!-- ko if: parent_url -->
```

Comment:
You can put these ko conditionals directly around the element you want to differ so that there is less repetition.